brokerapi/brokers/cloudsql/broker.go

```diff
@@ -421,20 +421,15 @@ func (b *CloudSQLBroker) Unbind(creds models.ServiceBindingCredentials) error {
 
 // gets the last operation for this instance and polls the status of it
 func (b *CloudSQLBroker) PollInstance(instanceId string) (bool, error) {
-	var op models.CloudOperation
-	var ii InstanceInformation
 	var instance models.ServiceInstanceDetails
 
 	if err := db_service.DbConnection.Where("id = ?", instanceId).First(&instance).Error; err != nil {
 		return false, models.ErrInstanceDoesNotExist
 	}
 
-	if err := json.Unmarshal([]byte(instance.OtherDetails), &ii); err != nil {
```

Comment:
Because it *should* be the same to look up by operation create time vs. operation id, and using create time is more consistent